sample.html

```diff
@@ -3,30 +3,75 @@
+  var vastUrl =
+    'https://pubads.g.doubleclick.net/gampad/ads?output=xml_vast3&env=vp&gdfp_req=1&unviewed_position_start=1&iu=%2F5441%2Fwka.life%2F_project43%2F%2Farticle%2Fpremium%2FFEATURED_VIDEO&sz=640x480&url=http%3A%2F%2Fproject43.wikia.com%2Fwiki%2FSyntheticTests%2FPremium%2FFeaturedVideo%3FAbTest.FEATURED_VIDEO_AUTOPLAY%3DCONTROL&correlator=1927378681583883&cust_params=wsi%3Doxap%26s0%3Dlife%26s0v%3Dlifestyle%26s0c%3Dtech%26s1%3D_project43%26s2%3Darticle%26ab%3D49_162%2C52_170%2C78_336%26ar%3D4%3A3%26artid%3D355%26dmn%3Dwikiacom%26hostpre%3Dproject43%26skin%3Doasis%26lang%3Den%26wpage%3Dsynthetictests%2Fpremium%2Ffeaturedvideo%26ref%3Ddirect%26esrb%3Dteen%26geo%3DPL%26pv%3D6%26u%3Dryo8cyllq%26ksgmnt%3D%26top%3D1k%26ah%3D9000%26pos%3DFEATURED_VIDEO%26src%3Dpremium&eid=41351088&sdkv=h.3.173.0&sdki=3c0d&scor=264486118542786&adk=3826960879&osd=2&frm=0&sdr=1&mpt=Ooyala&mpv=1.0&afvsz=200x200%2C250x250%2C300x250%2C336x280%2C450x50%2C468x60%2C480x70&ged=ve4_td164_tt158_pd164_la0_er419.141.574.441_vi0.0.941.1000_vp100_eb24171';
+  var vastUrl3 =
+    'https://pubads.g.doubleclick.net/gampad/ads?output=xml_vast3&env=vp&gdfp_req=1&unviewed_position_start=1&iu=%2F5441%2Fwka.life%2F_project43%2F%2Farticle%2Fpremium%2FFEATURED_VIDEO&sz=640x480&url=http%3A%2F%2Fpreview.project43.wikia.com%2Fwiki%2FDevTemplates%2FPremium%2FFeaturedVideo%2FWithSound&correlator=3343569300699169&cust_params=wsi%3Doxap%26s0%3Dlife%26s0v%3Dlifestyle%26s0c%3Dtech%26s1%3D_project43%26s2%3Darticle%26ab%3D49_162%2C52_170%26ar%3D3%3A4%26artid%3D369%26dmn%3Dwikiacom%26hostpre%3Dpreview%26skin%3Doasis%26lang%3Den%26wpage%3Ddevtemplates%2Fpremium%2Ffeaturedvideo%2Fwithsound%26ref%3Ddirect%26esrb%3Dteen%26geo%3DDE%26pv%3D5%26u%3Dry690akip%26ksgmnt%3D%26top%3D1k%26ah%3D9000%26pos%3DFEATURED_VIDEO%26src%3Dpremium&eid=324123020&sdkv=h.3.173.0&sdki=3c0d&scor=1618551238041221&adk=3826960879&osd=2&frm=0&sdr=1&mpt=Ooyala&mpv=1.0&afvsz=200x200%2C250x250%2C300x250%2C336x280%2C450x50%2C468x60%2C480x70&ged=ve4_td11_tt2_pd11_la11000_er435.102.590.402_vi0.0.927.921_vp100_eb24171';
   var playerParam = {
     "pcode": "5zb2wxOlZcNCe_HVT3a6cawW298X",
     "playerBrandingId": "26e2e3c1049c4e70ae08a242638b5c40",
     'debug':true,
     "skin": {
-      "config": "wikia.json"
+      "config": "wikia.json",
+      inline: {
+        adScreen: {
+          showControlBar: true,
+          forceShowAdControls: true
+        }
+      }
+    },
+    "autoplay": true, //Cookies.get('html5-skin.autoplay') !== '0',
```

Comment:
I think that "autoplay" should depend on this commented cookie instead of hardcoded true